ideas/100-smooth-chat-ux.md

```diff
@@ -0,0 +1,119 @@
+
+**V1 - quick fixes**
+- Chat view / message status
+- Chat view / last seen 
+- Chat view / user page 
+- Chat view / minior UI fixes 
+- Chat view / message space optimization
+- Chat view / timestamps
+- Chat view / adding/blocking new contact flow
+- Keyboard / use cases behaviors
+- Keyboard / maximum message width
+- Keyboard / fixing console position
+- Offline states and error handling 
+
+**V2 - group chat improvements**
+
+- Group chat view / chat metadata
+- Group chat view / message space optimization 
+- Group chat / timestamps
+- Group chat / seen by logic and documentation
+- Group chat / grouping messages 
+- Group chat view / top bar behavior
+
+**V3 - more engaging and visually clear home screen**
+
+- Home screen / starting a new chat flow
+- Home screen / visual consistency fixes
+- Home screen / search function 
+- Home screen / loading messages experience
+
+For more details about each one of those fixes, please visit: https://docs.google.com/document/d/1AyU9mWMG1MxNPPBr1kidIl7eMJzPhbUjJjUNBBxZpg8/edit
+
+### Team & Dependencies
+<!-- Are there bugs or feature requests in other repositories that are part of this Idea? -->
+<!-- There is no approval unless the idea requires to be reviewed by supporting organelles (Financial, Hiring, or Design). -->
+<!-- The Swarm must develop a fully fleshed out Requirements document for the idea to proceed, to the satisfaction of participants. -->
+Ideally, we would have at least 2 developers (Whisper, Clojure) in the team acting as a stakeholder in the process.
+
+Even though you won't be implementing those designs directly, you should act as if you were, so that what comes out of this process isn't a wishfull thinking of our collective imagination; bearing no relationship with reality (time constraints and limited resources). 
+
+<!--### Minimum Viable Product-->
+<!-- Mandatory, completes the Idea in the fastest route possible, can be hacky, needed to feel progress. See https://imgur.com/a/HVlw3 -->
+Goal Date: <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+
+### UX swarm, what's new: <!-- Description of Deliverables-->
+
+The idea behind the UX swarm to be able to work on the solutions (together) ahead of time; compared to how swarms have been working in the past. With this initiative we wish to achieve:
+
+- transparency about UX process; everyone is involved and welcome to join at any point
+- more collaboration between developers and designers 
+- be able to outsource some of the work (more bounties)
+- more time to fail, and try different solutions
+- better documentation and chance to validate assumptions (user testing)
+
+
+## Dates
+Goal Date:  <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+
+Description: <!-- Description of Deliverables-->
+
+Testing Days required: <!-- Days required at the end of development for testing -->
+
+## Success Metrics
+
+- average of 10 messages a day by each active user 
```

Comment:
Do we have baseline numbers for these states? If not, is this part of the initial scope perhaps? Would be useful from UXR POV, and might be some overlap with 68-core-metrics and 99-confidence